org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/NexusTest.java

```diff
@@ -179,7 +182,7 @@ public void createFile() throws IOException {
 	@After
 	public void deleteFile() {
 		try {
-			output.delete(); 
+			output.delete();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "delete". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS899)